example-config.json

```diff
@@ -66,7 +66,12 @@
         // Max number of requests to queue up before rejecting further requests.
         // Defaults to 0, which disables the limit.
         "max_queue_size": 500,
-        "admin_password": "xrp"
+        //if request contains header with authorization, clio will check if it matches this value's hash
```

Comment:
Let's keep the style of comments the same - Capitalize and add spaces where applicable 🙇